react-components/src/desktop/view/DesktopView.js

```diff
@@ -142,37 +144,46 @@ class DesktopView extends Component {
 
     handleMessage(event) {
         console.log(event.data);
-        let msg = null;
+        let push_msg = null;
         try {
-            msg = JSON.parse(event.data);
+            push_msg = JSON.parse(event.data);
         } catch (e) {
             return;
         }
-        if (msg.total) {
-            this.setState({unSeenCount: msg.total});
+        if (push_msg.total) {
+            this.setState({unSeenCount: push_msg.total});
+        }
+
+        let message = push_msg.message;
+        let category = message.type;
+        if (category === "team") {
+            message.message.text = message.message.text + message.payload.team_name;  // attach team name to team notifications.
         }
-        if (msg.message) {
+        if (message) {
```

Comment:
Here we're checking if there is a message, but on line 158 we're trying to get `message.type`.  Is there a possible NPE there?